search-handlers/src/test/java/no/unit/nva/indexing/handlers/IndexResourceHandlerTest.java

```diff
@@ -74,18 +73,17 @@ void shouldAddDocumentToIndexWhenResourceExistsInResourcesStorage() throws Excep
 
         InputStream input = createEventBridgeEvent(resourceLocation);
         indexResourceHandler.handleRequest(input, output, context);
-
-        verify(restHighLevelClient).index(any(), any());
+        Set<JsonNode> allIndexedDocuments = indexingClient.listAllDocuments(SAMPLE_RESOURCE.getIndexName());
+        assertThat(allIndexedDocuments, contains(SAMPLE_RESOURCE.getResource()));
```

Comment:
instead of verifying that the client was called, I verify that the indexdocument was actually indexed